File: 源码/EnjoySkin/app/src/main/java/com/enjoy/skin/MyApplication.java
package com.enjoy.skin;

import android.app.Application;

import com.enjoy.skin.lib.SkinManager;

/**
 * @author Lance
 * @date 2018/3/8
 */

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        SkinManager.init(this);

    }
}

File: projects_/LoggerProject/app/src/main/java/com/joyy/loggerproject/MyApplication.kt
package com.joyy.loggerproject

import android.app.Application
import com.joyy.log_core.KLog
import com.joyy.log_java.FileLog
import java.io.File

/**
 * Time:2021/7/29 12:27
 * Author: flannery
 * Description:
 */
class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        //KLog.initLog(LogOption(), FileLog())
        KLog.initLog(
            File(externalCacheDir, "logs"),
            FileLog()
        )
    }

    override fun onTerminate() {
        super.onTerminate()
        KLog.end()
    }
}